API.apib

```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
             ]
      }
   }
+
+# Get path from user location to specific store
+GET api/v1/tch/stores/:store_id/path{?address}
```

Comment:
should change to direction, can we have options for transportation

Reply:
Options for transportation are `driving`, `cycling` & `walking`. I thought that we agreed to use only `driving`

Reply:
yeah, currently only support driving but however in future we need to expand it